crates/state/src/wl_storage.rs

```diff
@@ -720,12 +730,24 @@ mod tests {
                 | Level::BlockWriteLog(WlMod::Delete | WlMod::DeletePrefix) => {
                 }
                 Level::TxWriteLog(WlMod::Write(val)) => {
+                    // NOTE: dummy WriteActions used here because this is
+                    // only for testing
+                    // TODO: may want to use test these new features though
+                    // in here.
```

Comment:
```suggestion
                    // NOTE: Will write to merkle tree and DB diff
```
I think this is fine, it's a private fn and we're only using it for `fn test_prefix_iters`. I don't think we need to test this in `wl_storage` specifically